backend/fregepoc/analyzers/tests/typescript/constants.py

```diff
@@ -0,0 +1,10 @@
+from fregepoc import settings
+
+MOCKED_TYPESCRIPT_FILES = (
```

Comment:
Why have you decided to change a default location of testing files? Most of the analyzers have their testing files in the `repositories` app. Also,  there has already  been defined a mock path for that location. I would recommend sticking to that convention, unless you have some reasons for not doing it.

Reply:
I believe the files you mean aren't meant for actual unit testing of analyzers, but rather downloading repos. I was following convention used in other analyzers where they keep files for unit tests in analogous location